SharpPcap/Tunneling/WinTap/WinTapDriver.cs

```diff
@@ -79,9 +80,8 @@ internal static void ConfigureDhcp(SafeFileHandle handle, IPAddress ip, IPAddres
         internal static void SetMediaStatus(SafeFileHandle handle, bool connected)
         {
             int value = connected ? 1 : 0;
-            Span<byte> inBuffer = stackalloc byte[4];
+            Span<byte> inBuffer = BitConverter.GetBytes(value);
```

Comment:
Why replace stack allocated bytes with heap allocation?

Reply:
I had to replace `MemoryMarshal.Write` since .NET 8 changed the API and requires it is called as `MemoryMarshal.Write(inBuffer, value);`

Reply:
The heap allocation is only for 4 bytes, and they gets cleaned using GC0 garbage collection

Reply:
what about:
```cs
#if NET8_0_OR_GREATER
            MemoryMarshal.Write(inBuffer, in value);
#else
            MemoryMarshal.Write(inBuffer, ref value);
#endif